src/YahooCalendar.ts

```diff
@@ -41,13 +41,13 @@ export default class YahooCalendar extends CalendarBase {
     if (this.isAllDay) {
       this
         .setParam('dur', 'allday')
-        .setParam('st', time.formatDate(this.start, FORMAT.DATE))
+        .setParam('st', time.formatDateNoUtc(this.start, FORMAT.DATE))
     } else {
-      this.setParam('st', time.formatDate(this.start, FORMAT.FULL))
+      this.setParam('st', time.formatDateNoUtc(this.start, FORMAT.FULL))
```

Comment:
`FORMAT.FULL` includes the `Z` at the end, which signifies that it's a UTC timestamp. It seems to work anyway (Yahoo just ignores it completely), but better safe to make a new constant for this

- [ ] add new const (`FORMAT.NO_UTC_FULL`?)